public/app/plugins/datasource/prometheus/configuration/AzureCredentialsForm.tsx

```diff
@@ -283,42 +224,6 @@ export const AzureCredentialsForm = (props: Props) => {
           )}
         </>
       )}
-      {getSubscriptions && (
```

Comment:
Why is the default subscription field being removed?

Reply:
Ignore this, just seen your comment in the PR description and I agree it looks like it's never been used 😊